axelor-account/src/main/java/com/axelor/apps/account/web/JournalController.java

```diff
@@ -85,4 +85,26 @@ public void toggleStatus(ActionRequest request, ActionResponse response) {
       TraceBackService.trace(response, e);
     }
   }
+
+  public void changeConfig(ActionRequest request, ActionResponse response) {
+    try {
+      Journal journal = request.getContext().asType(Journal.class);
+
```

Comment:
Could you declare a variable here to store journal.getCompany().getAccountConfig() and use it after? 
The code would be cleaner